src/tests/models/database/sequelize/sequelizeConnector.js

```diff
@@ -25,11 +27,23 @@ async function init(sequlizeClient) {
 }
 
 async function initSchemas() {
+    const file = client.define('file', {
+        id: {
+            type: Sequelize.DataTypes.UUID,
+            primaryKey: true
+        },
+        file: {
+            type: Sequelize.DataTypes.TEXT('long')
+        }
+    });
     const test = client.define('test', {
         test_id: {
             type: Sequelize.DataTypes.UUID,
             unique: 'compositeIndex'
         },
+        file_id: {
```

Comment:
if file_id is null, we want to delete it when we return the test. We don't want it to appear as `null` in our response when we call /tests/{{testId}}

Maybe insert it into the db only if it exists

need to fix this also for cassandra